go-controller/pkg/ovn/secondary_layer2_network_controller_test.go

```diff
@@ -283,11 +283,11 @@ var _ = Describe("OVN Multi-Homed pod operations for layer2 network", func() {
 			dummyLayer2PrimaryUserDefinedNetwork("192.168.0.0/16"),
 			nonICClusterTestConfiguration(),
 		),
-		table.Entry("pod on a user defined primary network on an interconnect cluster",
+		table.Entry("pod on a user defined primary network on an IC cluster",
 			dummyLayer2PrimaryUserDefinedNetwork("192.168.0.0/16"),
 			icClusterTestConfiguration(),
 		),
-		table.Entry("pod on a user defined primary network on an interconnect cluster",
+		table.Entry("pod on a user defined primary network on an IC cluster with per-pod SNATs disabled",
```

Comment:
nit ```suggestion
		table.Entry("pod on a user defined primary network on an IC cluster with per-pod SNATs enabled",
```
OR podCIDRSNAT disabled

Reply:
I'd prefer to keep the focus on what's enabled - not on what's disabled. 

I think it's easier to follow. 

... it's opinionated though.